scripts/build_fuzzers.py

```diff
@@ -0,0 +1,65 @@
+import subprocess
+import toml
+import typing
+import logging
+import tarfile
+import re
+import time
+
+REPO_DIR = '/home/runner/work/nearcore/nearcore/'
+G_BUCKET = 'gs://fuzzer_binaries/'
+ARCH_CONFIG_NAME = 'x86_64-unknown-linux-gnu'
+
+
+def get_archive_name(runner: str) -> str:
+    current_rev = time.strftime('%Y%m%d%H%M%S', time.gmtime())
+    tar_name = f'{runner}-{current_rev}.tar.gz'
+
+    logger.debug(f"archive name: {tar_name}")
+
+    return tar_name
+
+
+def push_to_google_bucket(runner: str) -> None:
+    tar_name = get_archive_name(runner)
+    with tarfile.open(name=tar_name, mode='w') as archiver:
+        archiver.add(f"target/{ARCH_CONFIG_NAME}/debug/{runner}", runner)
+
+    _proc = subprocess.run([
```

Comment:
do you want to use `check=True`? that way you get an exception if error occurs, otherwise we dont know when this fails.

Reply:
good catch, thank you!